swim/member.go

```diff
@@ -166,6 +187,23 @@ func (c Change) validateOutgoing() Change {
 	return c
 }
 
+func (c *Change) populateSubject(m *Member) {
```

Comment:
The `populateSubject` and `populateSource` methods feel a little bit like a constructor. Is it possible/feasible to add a constructor method (e.g. `newChange(subject, source *Member)`)?